spec.html

```diff
@@ -35566,8 +35583,8 @@ <h1>
         </emu-alg>
         <emu-grammar>AtomEscape :: `k` GroupName</emu-grammar>
         <emu-alg>
-          1. Search the enclosing |Pattern| for an instance of a |GroupSpecifier| containing a |RegExpIdentifierName| which has a CapturingGroupName equal to the CapturingGroupName of the |RegExpIdentifierName| contained in |GroupName|.
-          1. Assert: A unique such |GroupSpecifier| is found.
+          1. Let _matchingGroupSpecifiers_ be GroupSpecifiersThatMatch(|GroupName|).
+          1. Assert: _matchingGroupSpecifiers_ contains a single |GroupSpecifier|.
           1. Let _parenIndex_ be the number of left-capturing parentheses in the entire regular expression that occur to the left of the located |GroupSpecifier|. This is the total number of <emu-grammar>Atom :: `(` GroupSpecifier Disjunction `)`</emu-grammar> Parse Nodes prior to or enclosing the located |GroupSpecifier|, including its immediately enclosing |Atom|.
```

Comment:
`the located |GroupSpecifier|` is not specific enough. We can just pull out the first element of `matchingGroupSpecifiers` to its own alias and refer to that.

Reply:
Done.

The first 3 steps could be collapsed down to just:
```
Let _groupSpecifier_ be the sole element of
   GroupSpecifiersThatMatch(|GroupName|).
```
Let me know if you'd prefer that.